monai/bundle/scripts.py

```diff
@@ -625,8 +625,11 @@ def run(
 
     Args:
         run_id: ID name of the expected config expression to run, default to "run".
+            to run the config, the target config must contain this ID.
         init_id: ID name of the expected config expression to initialize before running, default to "initialize".
+            allow a config to have no `initialize` logic and the ID.
```

Comment:
```suggestion
            It's optional for both configs and `run`.
```

Reply:
Sure, updated in the latest commit, I didn't change the doc-string in the `ConfigWorkFlow`, the logic is slightly different.

Thanks.